data/src/main/java/org/oppia/android/data/backends/gae/api/PlatformParameterService.kt

```diff
@@ -0,0 +1,16 @@
+package org.oppia.android.data.backends.gae.api
+
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+/** Service that provides access to the Oppia platform parameter endpoint. */
+interface PlatformParameterService {
+
+// TODO(#3506): Change the url to point to the correct endpoint when the backend is ready
```

Comment:
2 spaces before this TODO (it should be aligned with other members of the class).

Reply:
Also, this should probably say something more like "Change the URL to point to the updated endpoint."

The current backend _is_ ready, we just want to change it.

Reply:
Done